libs/application/templates/signature-collection/municipal-list-signing/src/lib/dataSchema.ts

```diff
@@ -0,0 +1,7 @@
+import { z } from 'zod'
+
+export const dataSchema = z.object({
+  approveExternalData: z.boolean().refine((v) => v),
```

Comment:
This is fine, but I think you can also do `z.literal<boolean>(true)` if you prefer